dts/bindings/espi/microchip,xec-espi-saf-v2.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 Intel Corporation
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip ESPI SAF version 2 controller
+
+compatible: "microchip,xec-espi-saf-v2"
+
+include: espi-controller.yaml
+
+properties:
+    reg:
+      description: mmio register space
+      required: true
+
+    girqs:
+      type: array
+      required: true
+      description: Array of encoded interrupt information
+
+    pcrs:
+      type: array
+      required: true
+      description: Array of eSPI PCR register index and bit position
```

Comment:
A similar question here; why is this using a nonstandard `pcrs` property instead of using `reg`?

Reply:
every peripheral block has a sleep enable and clock enable bits, these are the PCR (power clock reset block) regs. The driver needs to know their respective sleep enable and clock enable bits.